working_demo.py

```
@@ -0,0 +1,671 @@
+#!/usr/bin/env python3
+"""
+Fixed Bitaxe Monitor Demo
+
+This version has working web interface that displays data correctly.
+"""
+
+import sys
+import os
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)